src/pages/Possibility/GivingTuesday.vue

```diff
@@ -109,6 +108,8 @@ $cta-color: #02582e;
 
 .cta-button {
 	@include button-style($cta-color, auto, #fff);
+	// Include in order to override the default box-shadow of this button
+	box-shadow: 0 2px darken($cta-color, 10%);
 }
```

Comment:
This was just a minor issue I saw while adding my component to the GivingTuesday page.